### DISCUSSION TITLE
No mount option

### DISCUSSION THREAD

**Discussion by 192709** [rustic-rs/rustic#915] [Things are not working as expected - is this a bug?] [answered]
Dear all,
from Documentation I got that I can mount a repository with "rustic -r /my/repository mount /my/mountpoint" 
, but if I do so I get 
error: unrecognized subcommand 'mount' .

rustic --help does **not** show  a mount subcommand either. 
I am on current version... v0.5.4.

Am I missing something?
Best Regards,
Marcus

PS:  fusermount is available

**Comment by aawsome** [comment, marked as answer]
HI @192709 - this is a bug in the docu. (The current docu is mainly copied from the restic docu and not yet fully adapted to what rustic is possible to do).
There is currently no `mount` command. Until it is implemented in rustic, please use restic for mounting a repository.

**Reply by 192709** [reply]
Thank you for explanation and providing a solution :+1:

**Comment by simonsan** [comment]
Fixed in: https://github.com/rustic-rs/docs/commit/bf6ddc65f44bba5c42b6b3f9abe1850750247e79
